OpenOversight/tests/conftest.py

```diff
@@ -62,8 +78,9 @@ def pick_birth_date():
 
 
 def pick_date(seed: bytes = None, start_year=2000, end_year=2020):
-    # source: https://stackoverflow.com/questions/40351791/how-to-hash-strings-into-a-float-in-01
-    # Wanted to deterministically create a date from a seed string (e.g. the hash or uuid on an officer object)
+    # source: https://stackoverflow.com/q/40351791
```

Comment:
I took the small link from the `Share` option, and now we have less link gore: 
<img width="944" alt="Screenshot 2023-07-10 at 10 16 04 AM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/6a025ebd-9339-4501-9e44-223d7e5bb007">

cc: @abandoned-prototype